mabout.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Meet Our Team</title>
+    <link rel="stylesheet" type="text/css" href="./css/about.css">
+</head>
+
+<body>
```

Comment:
please add <div class="overlay"></div> inside the body for the overlay to have effect for softening the background photo. These are already styled in styles.css.